inference/models/llama.cc

```diff
@@ -255,7 +257,8 @@ void LLAMA::create_llama_model(FFModel &ff,
       output = ff.sampling(softmax, generation_config.topp);
     } else {
       // output = ff.arg_top_k(dense, /*k=*/1, false);
-      output = ff.argmax(dense, /*beam_Search*/ false);
+      Tensor softmax = ff.softmax(dense, -1);
```

Comment:
@jiazhihao why is this change required?

Reply:
Because we used softmax cross entropy loss, so the last layer of the LLM must be softmax